Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -75,6 +77,8 @@ def bottle_args
              description: "Specify a committer name and email in `git`'s standard author format."
       flag   "--root-url=",
              description: "Use the specified <URL> as the root of the bottle's URL instead of Homebrew's default."
+      flag   "--root-url-using=",
+             description: "Use the specified download strategy for the root_url in the generated DSL via `using:' arg"
```

Comment:
```suggestion
             description: "Use the specified download strategy class for downloading the bottle's URL instead of Homebrew's default."
```

(and same below)